mobile/src/src/components/common/CommentBar/CommentBarButtons.jsx

```diff
@@ -52,31 +54,37 @@ const CommentBarButtons = ({
       >
         {/* Like button */}
         <View style={styles.buttonView}>
-          <IconButton
-            // TODO: Update design to match figma
-            icon={'heart'}
-            color={item.is_liked ? 'red' : 'black'}
+          <MaterialCommunityIcons
+            name={liked ? 'heart' : 'heart-outline'}
+            color={liked ? 'red' : 'purple'}
             style={styles.button}
             onPress={async () => {
-              await endpoint.like(item.id, !item.is_liked);
+              await endpoint.like(item.id, !liked);
               item.updateItem({
                 ...item,
-                is_liked: !item.is_liked,
-                likes: item.is_liked ? item.likes - 1 : item.likes + 1,
+                is_liked: !liked,
+                likes: liked ? item.likes - 1 : item.likes + 1,
               });
             }}
-          />
+          ></MaterialCommunityIcons>
         </View>
         {/* Button to view comments */}
-        <View>
-          <IconButton
-            icon={'comment'}
-            color={'black'}
+        <View style={styles.buttonView}>
+          <MaterialCommunityIcons
+            name="chat-outline"
+            color={'purple'}
             style={styles.button}
             onPress={() => {
               setSendButtonVisible(true);
             }}
-          />
+          ></MaterialCommunityIcons>
+        </View>
+        <View style={styles.buttonView}>
+          <MaterialCommunityIcons
+            name="bookmark-outline"
+            style={styles.button}
+            color={'purple'}
+          ></MaterialCommunityIcons>
```

Comment:
No need for separate closing tab

Reply:
When I save using prettier it default's to this type of closing tabs